lib/model.js

```diff
@@ -990,9 +968,8 @@ Model.prototype.removeAttribute = function(attribute) {
  * @return {Promise<this>}
  */
 Model.prototype.sync = function(options) {
-  options = options || {};
+  options = _.extend({}, this.options, options);
```

Comment:
Is it possible that `this.options.hooks` could be defined? If so, this needs to be: `options = _.extend({}, this.options, options, {hooks: options.hooks === undefined ? true : !!options.hooks})` to maintain exact same behaviour.